### PR TITLE
Allowing user to create pool pairs

### PR DESCRIPTION
**Pull request by fakoredeDamilola**
Allow user to create a token pair and add liquidity to new pairs

### CODE REVIEW

**Comment by fakoredeDamilola** [comment]
This PR allow users create a new pool with new addresses
https://www.loom.com/share/aadc80059e5542db88e2e88bd0bf3400

File: app/containers/App/index.js [hunk 1]
```diff
@@ -74,21 +74,26 @@ const App = props => {
       method: 'eth_chainId',
     })
     props.updateChainId(chainID)
-    if (isSupportedNetwork(chainID)) {
-      listener(wallet, props);
-      reConnector(props);
+    if (!isSupportedNetwork(chainID)) {
```
Comment (kayroy247) [inline app/containers/App/index.js:77]: Is this also a delibrate change.

**Review by kayroy247 — COMMENTED** [review]
Good job with the create pair feature. However, kindly check all changes to ensure you only have the changes you made on this PR.

And also resolve the merge conflict on the PR.

File: app/containers/WalletProvider/actions.js [hunk 2]
```diff
@@ -141,11 +139,8 @@ export const changeRGPValue = wallet => async dispatch => {
...
 export const getTokenAddress = (chainId) => {
-  if (chainId === '0x38' && window.ethereum !== undefined && window.ethereum.isMetaMask) {
+  if (chainId === '0x38') {
     return '0xFA262F303Aa244f9CC66f312F0755d89C3793192';
   }
-  if (chainId === '0x61') {
-    return '0x9f0227a21987c1ffab1785ba3eba60578ec1501b';
-  }
-  return window.ethereum !== undefined && window.ethereum.isTrust && chainId == '0x38' && '0xFA262F303Aa244f9CC66f312F0755d89C3793192';
+  return '0x9f0227a21987c1ffab1785ba3eba60578ec1501b';
```
[the remark this answers is not in the document]
Comment (fakoredeDamilola) [reply app/containers/WalletProvider/actions.js:145]: resolved

**Pull request merged**